src/email.rs

```diff
@@ -37,19 +34,17 @@ fn build_email(
     subject: &str,
     body: &str,
     mailgun_config: &Option<MailgunConfigVars>,
-) -> AppResult<SendableEmail> {
+) -> AppResult<Message> {
     let sender = mailgun_config
         .as_ref()
         .map(|s| s.smtp_login.as_str())
         .unwrap_or("test@localhost");
 
-    let email = Email::builder()
-        .to(recipient)
-        .from(sender)
+    let email = Message::builder()
+        .to(recipient.parse().unwrap())
+        .from(sender.parse().unwrap())
```

Comment:
This should fix the `sending_to_invalid_email_fails` test
```suggestion
        .to(recipient.parse()?)
        .from(sender.parse()?)
```